glide.lock

```diff
@@ -271,6 +271,7 @@ imports:
   - kubernetes/typed/storage/v1/fake
   - kubernetes/typed/storage/v1beta1
   - kubernetes/typed/storage/v1beta1/fake
+  - pkg/api/v1
```

Comment:
Np, though I seem to have confused this PR with the other, oops :). This one isn't pulling pkg/volume/util it's pulling events for a single const, in this case we may as well redefine the const here since there's no value added from vendoring the extra code (unless you count extra bytes in `.git` )